tests/main/snapshot-basic/task.yaml

```diff
@@ -81,6 +72,22 @@ execute: |
     snap forget "$SET_ID"
     snap saved --id="$SET_ID" | grep "No snapshots found"
 
+    echo "Check that import works"
+    RESTORE_ID=$( snap import-snapshot "${SET_ID}_export.snapshot" | head -n1 | cut -f2 -d'#' )
+    echo "And is valid"
+    snap check-snapshot "$RESTORE_ID"
+
+    NUM_SAVED=$(snap saved | wc -l)
+
+    echo "Import again doesn't create a duplicated snapshot"
+    IMPORTED_ID=$( snap import-snapshot "${SET_ID}_export.snapshot" | head -n1 | cut -f2 -d'#' )
```

Comment:
could this use `SET_ID}_export-2.snapshot` for extra robustness? It seems to me that since we exported the same snapshot to two different files (which should be the same file too), we should get a duplicated set ID if we use the second file same as using the first file

Reply:
Yes, makes sense to do it that way, thanks.